MMM-Bring.js

```diff
@@ -127,6 +140,11 @@ Module.register("MMM-Bring", {
         return container;
     },
 
+    openKeyboard: function() {
+        console.log("MMM-Bring opening keyboard");
```

Comment:
Should we use logger instead?

Reply:
Sorry, What do you mean with logger?

I usually use a debug logger function in my modules:
```
  log: function (msg) {
    if (this.config && this.config.debug) {
      console.log(this.name + ": ", (msg));
    }
  }
```
Can implement this here if you like.

Reply:
Or do you mean `log.info(msg)`?

Reply:
There is a Log-Object available for logging.
e.g. Log.info("test") so the log-output appears in the MM-log and not the browser window. What do you say? console.log is also fine for me, it's just that I've seen other projects using this instead

Reply:
I thought `Log.info` goes into stderr and hence into the devtools as well as `console.log`?
https://github.com/MichMich/MagicMirror/wiki/Module-Debugging#logging-from-mmm-your-modulejs.

But I can change it of course. It's your repository :-)

Reply:
```
var Log = (function() {
	return {
		info: Function.prototype.bind.call(console.info, console),
		log:  Function.prototype.bind.call(console.log, console),
```

Reply:
Ok good to know thanks...